### DISCUSSION TITLE
[Q] workbook.calcProperties.fullCalcOnLoad = true doesn't seem to work

### DISCUSSION THREAD

**Discussion by LOUKOUKOU** [exceljs/exceljs#2419] [Q&A]
## 💬 Questions and Help
So with the following code:

```
const workbook = new Excel.Workbook()
await workbook.xlsx.readFile('file_a.xlsx')
workbook.calcProperties.fullCalcOnLoad = true
console.log(workbook.getWorksheet('Sheet1').getCell('A1').value) // 5
console.log(workbook.getWorksheet('Sheet1').getCell('A2').value) // { formula: 'SUM(A1,3)', result: 8 }
workbook.getWorksheet('Sheet1').getCell('A1').value = 8
await workbook.xlsx.writeFile('file_b.xlsx')

await workbook.xlsx.readFile('file_b.xlsx')
console.log(workbook.getWorksheet('Sheet1').getCell('A2').value) // { formula: 'SUM(A1,3)', result: 8 }, but it should be 11
```

Notice how the value doesn't change. The strange thing is that when I open up the document in Excel, it does show that it changed, but its only if I save it in Excel that it reflex in code. Am I doing something wrong?

**Comment by LOUKOUKOU** [comment]
Never mind, this approach won't work, because for the values to update you would have to open up the file manually, or use some funky Shell scripts.

**Comment by chuchu1313** [comment]
Hi @LOUKOUKOU , I also face this issue. Could you please share how do you solve this.
